client/src/components/WorkflowInvocationState/util.ts

```diff
@@ -1,11 +1,28 @@
-import type { InvocationJobsSummary } from "@/api/invocations";
+import type { InvocationJobsSummary, StepJobSummary } from "@/api/invocations";
 import { ERROR_STATES, NON_TERMINAL_STATES, TERMINAL_STATES } from "@/api/jobs";
 
+import type { ReasonToLevel } from "./types";
+
 export { ERROR_STATES, NON_TERMINAL_STATES, TERMINAL_STATES } from "@/api/jobs";
 
 export const POPULATED_STATE_FAILED = "failed";
 
-function countStates(jobSummary: InvocationJobsSummary | null, queryStates: string[]): number {
+export const INVOCATION_MSG_LEVEL: ReasonToLevel = {
+    history_deleted: "cancel",
```

Comment:
Is there a way to not duplicate the dictionary defined in the types.d.ts file above? Will I get a type error at least if I update one but not the other?

Reply:
Good catch! I've tried out this approach which I reckon should work way better:

<img width="1420" height="486" alt="image" src="https://github.com/user-attachments/assets/b0ff557d-8bf8-436a-9678-cab85f17fc9a" />

<img width="748" height="493" alt="image" src="https://github.com/user-attachments/assets/f3f73a22-f113-4d3a-b731-6b3efd489ba9" />